src/styles/product-card.scss

```
@@ -11,9 +11,17 @@
   }
 
   &__image {
-    width: 160px;
-    height: 134px;
+    width: 166px;
+    height: 150px;
     margin: 0 auto 45px;
+    padding: 16px 3px 0;
+    transition: transform 1s;
+    &:hover {
+      transform: scale(1.2);
+      border: 3px solid #f3f3f3;
+      border-radius: 5px;
```

буде красивіше коли тільки верхні кути заокруглити. Хоча я цього не задавав на початку взагалі поки мені не написали, бо в завданні не вказано))
```suggestion
      border-radius: 5px 5px 0 0;
```